src/BrowserStorageProvider.ts

```diff
@@ -0,0 +1,28 @@
+import type { StorageProvider } from './types/index.js'
+
+export const BrowserStorageProvider: StorageProvider = {
```

Comment:
We are only providing storage provider for non-node because that's the only one we currently need.